_posts/brianna-stedman/2020-12-02-brianna-stedman-milestones-3.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+author: brianna-stedman
+title: Brianna's final project milestones, last revision
```

Comment:
Would you mind enclosing your title in `"`s? Some characters are invalid outside of a string, and I think `'` is one of them.

Reply:
shoot, good catch, I'll update it!

Reply:
updated, I think it's ready to merge :-) thank you so much!